src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/projects/ProjectSampleMetadataImportPageIT.java

```diff
@@ -89,4 +90,11 @@ public void testFailedUpload() {
 		page.goToReviewPage();
 		assertTrue(page.isAlertDisplayed(), "Validation message did not display");
 	}
+
+	@Test
+	public void testDuplicateHeaders() {
+		ProjectSampleMetadataImportPage page = ProjectSampleMetadataImportPage.goToPage(driver());
+		page.uploadMetadataFile(DUPLICATE_HEADERS_FILE_PATH);
+		assertTrue(page.isAlertDisplayed(), "Validation message did not display");
+	}
```

Comment:
Add a test for displaying the validation error alert when there are duplicate headers as well.

Reply:
I'm confused, this is a test for displaying the alert when there are duplicate headers.

Reply:
My bad. The message just needs to be updated. Currently it says "Validation messages did not display"

Reply:
Disregard this comment